cypress-tests/integration-tests/features/report-list-render-handler.feature

```diff
@@ -52,3 +52,4 @@ Feature: List report
   Scenario: Navigating paging
     When I click a paging link
     Then the current page is shown in the URL
```

Comment:
Please could you add some integration test coverage for this? I know the coverage is a bit haphazard at the moment (and by all means keep me honest in my PRs too), but it would be good to add it where possible.